app/helpers/labware_helper.rb

```diff
@@ -27,12 +27,12 @@ def self.cycling_colours(name, &block)
   cycling_colours(:bait)    { |labware, _|            labware.bait }
   cycling_colours(:pooling) { |_labware, destination| destination }
 
-  def permanent_state(container)
-    container.state == 'failed' ? 'failed' : 'good'
+  def failable?(container)
+    container.passed? && container.control_info != 'negative'
   end
 
-  def failable?(container)
-    container.state == 'passed'
+  def prevent_quadrant_fail?(container)
+    !container.passed?
```

Comment:
Two methods here, one to indicate that a well can be failed individually, one to let it be failed en-mass. (You can't call fail on an already failed well)

Reply:
I wasn't 100% comfortable with sticking this logic in a helper. Along with the control_info_formatted method I'm wondering if we need a lightweight presenter for wells?